testgrid/tgrun/pkg/scheduler/types/types.go

```diff
@@ -6,15 +6,17 @@ import (
 )
 
 type SchedulerOptions struct {
-	APIEndpoint  string
-	APIToken     string
-	OverwriteRef bool
-	Ref          string
-	Staging      bool
-	Airgap       bool
-	KurlVersion  string
-	Spec         string
-	OSSpec       string
+	APIEndpoint       string
+	APIToken          string
+	OverwriteRef      bool
+	Ref               string
+	Staging           bool
+	Airgap            bool
+	KurlVersion       string
+	Spec              string
+	OSSpec            string
+	NumPrimaryNodes   int
```

Comment:
remove this from here

Reply:
This is actually required, the plan is to feed this number using this command
```
tgrun queue --os-spec hack/os-spec.yaml --spec hack/test-spec.yaml --ref test-6  --num-primary-node 1 --num-secodady-node 0
```
where 
the default for primary =1

and secondary =0

then we can consume them while we are running the ``tgrun run``  command